gradle.properties

```diff
@@ -1,4 +1,4 @@
 group = org.controlsfx
 controlsfx_version = 11.1.3-SNAPSHOT
 fxsampler_version = 1.0.12-SNAPSHOT
-javafx_version = 13
\ No newline at end of file
+javafx_version = 21
```

Comment:
I would keep this as either 13 or move to the last LTS release i.e. 17

Reply:
Isn't 21 LTS as well?  At least that's what I read at https://gluonhq.com/products/javafx/
 The key issue was that it was a clash with javafx 21, therefore it makes sense to have this version here.

Reply:
Yes, it is. However, ControlsFX is used across many applications and not everyone would move to latest version immediately.  The changes done is this PR makes sure that ControlsFX works with 21, but it also works with versions before 21. That's what I like about this PR :)

My personal preference is to keep the compatibility version as minimum as possible, so that more applications can use the library without having to move to the latest JavaFX version.